### ISSUE TITLE
[Usability]A11y_PowerShellEditorServices_Terminal_Usability : After hitting on the "kill terminal" button error messege appears in pop up but screen reader does not takes focus of the controls of the popup it announce error messege dynamically.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#62033]
Environment Details:
VSCode Version : 1.28.2
OS Version : Win10

Additional information : Usability issue

Repro Steps:
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate "Terminal panel" & press enter 
4.Navigate to" Kill button " & press enter.
5.Start NVDA & check whether it is reading proper error message or not.
 
Actual:
After hitting on the "kill terminal" button error message appears in pop up but screen reader does not takes focus of the controls of the popup it announce error message dynamically.

Expected:
After hitting on the "kill terminal" button error message appears in pop up but screen reader should takes focus of the controls of the popup it announce error message dynamically 
 
User Impact:
Screen reader users finds difficult to understand the functionality as well unable to navigate on controls of the error popup

Does this issue occur when all extensions are disabled?: Yes
Attachment:
[A11y_PowerShellEditorServices_Terminal_Usability11.pptx](https://github.com/Microsoft/vscode/files/2525006/A11y_PowerShellEditorServices_Terminal_Usability11.pptx)

**Comment by Tyriar** [comment]
@bpasero possibly a problem with the notification? How does it alert when popped up?

**Comment by bpasero** [comment]
Using the `alert` `role`:

https://github.com/Microsoft/vscode/blob/57d85c2667e8e66140e0f7edfb8b6e636c6a4876/src/vs/workbench/browser/parts/notifications/notificationsAlerts.ts#L70 

We cannot give focus to the notification obviously when it appears.
